nomenclature/core.py

```diff
@@ -7,10 +7,13 @@
 class Nomenclature:
     """A nomenclature with codelists for all dimensions used in the IAMC data format"""
 
-    def __init__(self, path="definitions"):
+    def __init__(self, path):
         if not isinstance(path, Path):
             path = Path(path)
 
+        if not path.is_dir():
+            raise FileNotFoundError(f"Definitions directory not found: {path}")
```

Comment:
Maybe change to [`NotADirectoryError`](https://docs.python.org/3/library/exceptions.html#NotADirectoryError) since we're specifically looking for a directory. Maybe I'm also being overly knit-picky, in this case ignore ;)

Reply:
👍

Reply:
wonderful